openpype/hosts/houdini/plugins/create/create_pointcache.py

```diff
@@ -19,6 +19,10 @@ def create(self, subset_name, instance_data, pre_create_data):
         instance_data.pop("active", None)
         instance_data.update({"node_type": "alembic"})
 
+        if not instance_data.get("families"):
+            instance_data["families"] = []
```

Comment:
When would this occur? If this happens, doesn't that also mean it doesn't have `pointcache` in the families and thus we've a broken instance, or?

Reply:
I am paranoid. Couldn't find any point in code that is actually forcing "families" as list on `instance_data`

Reply:
I see, but _if_ it doesn't enforce it - shouldn't it _also_ contain `pointcache` (or add it?)

Reply:
Well since it is already in `family`, it doesn't have to be in `families`.  With this, it won't crash for artist because it doesn't have to. Of course, question is what is also missing if `families` is not there. I could just do:

```python
instance_data["families"] = ["abc"]
```
but what if logic calling `create()` will change and will start to add something there (not sure why though :)) So this was reasonably safe approach to it - append.